src/view.js

```diff
@@ -0,0 +1,42 @@
+import Vue from 'vue'
+import * as library from './library'
+
+Vue.config.productionTip = false
+
+/* eslint-disable no-new */
+const vm = new Vue({
+    el: '#app',
+    data () {
+        return {
+            componentName: null,
+            props: {},
+        }
+    },
+    render (h) {
+        let component = library[this.componentName]
+        if (!component) {
+            component = Object.values(library).find(x => x.name === this.componentName)
+        }
+
+        if (!component) {
```

Comment:
Is this valid?

Reply:
Maybe we could throw an exception instead.

Reply:
Yes, probably better